src/stories/colour-tokens.d.ts

```diff
@@ -0,0 +1,82 @@
+export const ColourBlack: string;
```

Comment:
This file is taken from the token build process from this work in progress PR
https://github.com/wellcometrust/design-system/pull/2

Reply:
Is there a more concise way of writing these? e.g. creating an iterator of colour/number values and looping it, rather than declaring each individual variant - seems cumbersome as is, and requires us to manually declare each new variant which might be added further down the line

Reply:
It was a quick fix to get over a problem which I've since added a fix for i.e. `Object.entries(tokens).map(([key, value]) ...` will correctly assign types for key and value.